lighthouse-cli/bin.ts

```diff
@@ -67,6 +67,7 @@ const cliFlags = yargs
     'mobile',
     'save-assets',
     'save-artifacts',
+    'exclude-artifacts',
```

Comment:
I guess the question is, what does this flag observably *do*? At this point it won't affect the html report, so users need to choose json to see the artifacts in the output (and they could use `--save-artifacts` for that).

Maybe it makes sense not to have a CLI flag right now, always clear `results.artifacts` in bin.ts, and have future PRs (like #1130 adding `--view`) add the flags controlling if the `artifacts` property isn't cleared.

That should help with later orthogonality of flags, so we don't have to figure out things like, what if the user sets `--view` *and* `--include-artifacts=false`